From c62a1af0833145990f79eb34817f90d8e48cb7bc Mon Sep 17 00:00:00 2001
From: Ian Chien <ianchien.wiwynn@gmail.com>
Date: Fri, 19 Jul 2024 15:59:45 +0800
Subject: [PATCH 13/22] configurations: yosemite4: Remove duplicate Fan Board
 configs

This commit removes the duplicate fan board configs since it should only
load one config for one FRU instead of loading multiple configs.
Test Plan: Fan board sensor config can be loaded and fan board sensors
can be read.

Change-Id: I0c06b747671d6b59d0942c327e4782c556390282
Signed-off-by: Ian Chien <ianchien.wiwynn@gmail.com>
---
 configurations/yosemite4_fanboard_adc_ti.json | 507 ----------
 .../yosemite4_fanboard_fsc_max.json           | 890 ------------------
 .../yosemite4_fanboard_fsc_max_adc_ti.json    |   4 +-
 .../yosemite4_fanboard_fsc_nct.json           | 889 -----------------
 meson.build                                   |   3 -
 5 files changed, 2 insertions(+), 2291 deletions(-)
 delete mode 100644 configurations/yosemite4_fanboard_adc_ti.json
 delete mode 100644 configurations/yosemite4_fanboard_fsc_max.json
 delete mode 100644 configurations/yosemite4_fanboard_fsc_nct.json

diff --git a/configurations/yosemite4_fanboard_adc_ti.json b/configurations/yosemite4_fanboard_adc_ti.json
deleted file mode 100644
index cf1da3e..0000000
--- a/configurations/yosemite4_fanboard_adc_ti.json
+++ /dev/null
@@ -1,507 +0,0 @@
-{
-    "Exposes": [
-        {
-            "Address": "0x1f",
-            "Bus": 30,
-            "Labels": [
-                "in0",
-                "in1",
-                "in2",
-                "in3",
-                "in4",
-                "in5",
-                "in6",
-                "in7"
-            ],
-            "Name": "FANBOARD0_ADC",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Label": "in0",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in0",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in0",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in1",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in1",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in1",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in2",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in2",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in2",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in3",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in3",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in3",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in4",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in4",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in4",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in5",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in5",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in5",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in6",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 60
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in6",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 53.28
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in6",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 52.8
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in6",
-                    "Name": "lower non critical",
-                    "Severity": 0,
-                    "Value": 43.2
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in6",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 42.72
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in6",
-                    "Name": "lower non recoverable",
-                    "Severity": 4,
-                    "Value": 40
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in7",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 3.83
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in7",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 3.432
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in7",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 3.399
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in7",
-                    "Name": "lower non critical",
-                    "Severity": 0,
-                    "Value": 3.201
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in7",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 3.168
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in7",
-                    "Name": "lower non recoverable",
-                    "Severity": 4,
-                    "Value": 2.64
-                }
-            ],
-            "Type": "ADC128D818",
-            "in0_Name": "FANBOARD0_ADC_48V_FAN8_CURR_A",
-            "in0_Scale": 300,
-            "in1_Name": "FANBOARD0_ADC_48V_FAN4_CURR_A",
-            "in1_Scale": 300,
-            "in2_Name": "FANBOARD0_ADC_48V_FAN0_CURR_A",
-            "in2_Scale": 300,
-            "in3_Name": "FANBOARD0_ADC_48V_FAN9_CURR_A",
-            "in3_Scale": 300,
-            "in4_Name": "FANBOARD0_ADC_48V_FAN5_CURR_A",
-            "in4_Scale": 300,
-            "in5_Name": "FANBOARD0_ADC_48V_FAN1_CURR_A",
-            "in5_Scale": 300,
-            "in6_Name": "FANBOARD0_ADC_48V_AUX_VOLT_V",
-            "in6_Scale": 31.9458,
-            "in7_Name": "FANBOARD0_ADC_3V3_STBY_VOLT_V",
-            "in7_Scale": 500
-        },
-        {
-            "Address": "0x1f",
-            "Bus": 31,
-            "Labels": [
-                "in0",
-                "in1",
-                "in2",
-                "in3",
-                "in4",
-                "in5",
-                "in6",
-                "in7"
-            ],
-            "Name": "FANBOARD1_ADC",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Label": "in0",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in0",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in0",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in1",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in1",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in1",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in2",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in2",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in2",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in3",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in3",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in3",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in4",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in4",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in4",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in5",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 6
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in5",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 2.156
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in5",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 2.058
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in6",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 60
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in6",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 53.28
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in6",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 52.8
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in6",
-                    "Name": "lower non critical",
-                    "Severity": 0,
-                    "Value": 43.2
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in6",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 42.72
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in6",
-                    "Name": "lower non recoverable",
-                    "Severity": 4,
-                    "Value": 40
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in7",
-                    "Name": "upper non recoverable",
-                    "Severity": 4,
-                    "Value": 3.83
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in7",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 3.432
-                },
-                {
-                    "Direction": "greater than",
-                    "Label": "in7",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 3.399
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in7",
-                    "Name": "lower non critical",
-                    "Severity": 0,
-                    "Value": 3.201
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in7",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 3.168
-                },
-                {
-                    "Direction": "less than",
-                    "Label": "in7",
-                    "Name": "lower non recoverable",
-                    "Severity": 4,
-                    "Value": 2.64
-                }
-            ],
-            "Type": "ADC128D818",
-            "in0_Name": "FANBOARD1_ADC_48V_FAN10_CURR_A",
-            "in0_Scale": 300,
-            "in1_Name": "FANBOARD1_ADC_48V_FAN6_CURR_A",
-            "in1_Scale": 300,
-            "in2_Name": "FANBOARD1_ADC_48V_FAN2_CURR_A",
-            "in2_Scale": 300,
-            "in3_Name": "FANBOARD1_ADC_48V_FAN11_CURR_A",
-            "in3_Scale": 300,
-            "in4_Name": "FANBOARD1_ADC_48V_FAN7_CURR_A",
-            "in4_Scale": 300,
-            "in5_Name": "FANBOARD1_ADC_48V_FAN3_CURR_A",
-            "in5_Scale": 300,
-            "in6_Name": "FANBOARD1_ADC_48V_AUX_VOLT_V",
-            "in6_Scale": 31.9458,
-            "in7_Name": "FANBOARD1_ADC_3V3_STBY_VOLT_V",
-            "in7_Scale": 500
-        }
-    ],
-    "Name": "Yosemite 4 Fan Board $bus % 30 ADC",
-    "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'FAN Board FSC-.* ADC-TI', 'PRODUCT_PRODUCT_NAME': 'Yosemite V4'})",
-    "Type": "Board",
-    "xyz.openbmc_project.Inventory.Decorator.Asset": {
-        "Manufacturer": "$PRODUCT_MANUFACTURER",
-        "Model": "$PRODUCT_PRODUCT_NAME",
-        "PartNumber": "$PRODUCT_PART_NUMBER",
-        "SerialNumber": "$PRODUCT_SERIAL_NUMBER"
-    },
-    "xyz.openbmc_project.Inventory.Decorator.AssetTag": {
-        "AssetTag": "$PRODUCT_ASSET_TAG"
-    },
-    "xyz.openbmc_project.Inventory.Decorator.Revision": {
-        "Version": "$PRODUCT_VERSION"
-    }
-}
diff --git a/configurations/yosemite4_fanboard_fsc_max.json b/configurations/yosemite4_fanboard_fsc_max.json
deleted file mode 100644
index 7a13535..0000000
--- a/configurations/yosemite4_fanboard_fsc_max.json
+++ /dev/null
@@ -1,890 +0,0 @@
-{
-    "Exposes": [
-        {
-            "Address": "$address",
-            "Bus": "$bus",
-            "Name": "Fan Board $bus % 30 FRU",
-            "Type": "EEPROM"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN0_TACH_IL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD0_FAN0_PWM",
-                "Tachs": [
-                    3
-                ]
-            },
-            "Index": 3,
-            "Name": "FANBOARD0_FAN0_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN0_TACH_OL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD0_FAN0_PWM",
-                "Tachs": [
-                    10
-                ]
-            },
-            "Index": 10,
-            "Name": "FANBOARD0_FAN0_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN1_TACH_IL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD0_FAN1_PWM",
-                "Tachs": [
-                    3
-                ]
-            },
-            "Index": 3,
-            "Name": "FANBOARD0_FAN1_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN1_TACH_OL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD0_FAN1_PWM",
-                "Tachs": [
-                    10
-                ]
-            },
-            "Index": 10,
-            "Name": "FANBOARD0_FAN1_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN2_TACH_IL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD1_FAN2_PWM",
-                "Tachs": [
-                    3
-                ]
-            },
-            "Index": 3,
-            "Name": "FANBOARD1_FAN2_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN2_TACH_OL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD1_FAN2_PWM",
-                "Tachs": [
-                    10
-                ]
-            },
-            "Index": 10,
-            "Name": "FANBOARD1_FAN2_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN3_TACH_IL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD1_FAN3_PWM",
-                "Tachs": [
-                    3
-                ]
-            },
-            "Index": 3,
-            "Name": "FANBOARD1_FAN3_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN3_TACH_OL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD1_FAN3_PWM",
-                "Tachs": [
-                    10
-                ]
-            },
-            "Index": 10,
-            "Name": "FANBOARD1_FAN3_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN4_TACH_IL",
-                "Pwm": 1,
-                "PwmName": "FANBOARD0_FAN4_PWM",
-                "Tachs": [
-                    2
-                ]
-            },
-            "Index": 2,
-            "Name": "FANBOARD0_FAN4_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN4_TACH_OL",
-                "Pwm": 1,
-                "PwmName": "FANBOARD0_FAN4_PWM",
-                "Tachs": [
-                    9
-                ]
-            },
-            "Index": 9,
-            "Name": "FANBOARD0_FAN4_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN5_TACH_IL",
-                "Pwm": 1,
-                "PwmName": "FANBOARD0_FAN5_PWM",
-                "Tachs": [
-                    2
-                ]
-            },
-            "Index": 2,
-            "Name": "FANBOARD0_FAN5_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN5_TACH_OL",
-                "Pwm": 1,
-                "PwmName": "FANBOARD0_FAN5_PWM",
-                "Tachs": [
-                    9
-                ]
-            },
-            "Index": 9,
-            "Name": "FANBOARD0_FAN5_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN6_TACH_IL",
-                "Pwm": 1,
-                "PwmName": "FANBOARD1_FAN6_PWM",
-                "Tachs": [
-                    2
-                ]
-            },
-            "Index": 2,
-            "Name": "FANBOARD1_FAN6_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN6_TACH_OL",
-                "Pwm": 1,
-                "PwmName": "FANBOARD1_FAN6_PWM",
-                "Tachs": [
-                    9
-                ]
-            },
-            "Index": 9,
-            "Name": "FANBOARD1_FAN6_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN7_TACH_IL",
-                "Pwm": 1,
-                "PwmName": "FANBOARD1_FAN7_PWM",
-                "Tachs": [
-                    2
-                ]
-            },
-            "Index": 2,
-            "Name": "FANBOARD1_FAN7_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN7_TACH_OL",
-                "Pwm": 1,
-                "PwmName": "FANBOARD1_FAN7_PWM",
-                "Tachs": [
-                    9
-                ]
-            },
-            "Index": 9,
-            "Name": "FANBOARD1_FAN7_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN8_TACH_IL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD0_FAN8_PWM",
-                "Tachs": [
-                    0
-                ]
-            },
-            "Index": 0,
-            "Name": "FANBOARD0_FAN8_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN8_TACH_OL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD0_FAN8_PWM",
-                "Tachs": [
-                    1
-                ]
-            },
-            "Index": 1,
-            "Name": "FANBOARD0_FAN8_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN9_TACH_IL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD0_FAN9_PWM",
-                "Tachs": [
-                    0
-                ]
-            },
-            "Index": 0,
-            "Name": "FANBOARD0_FAN9_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN9_TACH_OL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD0_FAN9_PWM",
-                "Tachs": [
-                    1
-                ]
-            },
-            "Index": 1,
-            "Name": "FANBOARD0_FAN9_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN10_TACH_IL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD1_FAN10_PWM",
-                "Tachs": [
-                    0
-                ]
-            },
-            "Index": 0,
-            "Name": "FANBOARD1_FAN10_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x20",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN10_TACH_OL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD1_FAN10_PWM",
-                "Tachs": [
-                    1
-                ]
-            },
-            "Index": 1,
-            "Name": "FANBOARD1_FAN10_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN11_TACH_IL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD1_FAN11_PWM",
-                "Tachs": [
-                    0
-                ]
-            },
-            "Index": 0,
-            "Name": "FANBOARD1_FAN11_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x2f",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN11_TACH_OL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD1_FAN11_PWM",
-                "Tachs": [
-                    1
-                ]
-            },
-            "Index": 1,
-            "Name": "FANBOARD1_FAN11_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        }
-    ],
-    "Name": "Yosemite 4 Fan Board $bus % 30 FSC",
-    "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'FAN Board FSC-MAX ADC-.*', 'PRODUCT_PRODUCT_NAME': 'Yosemite V4'})",
-    "Type": "Board",
-    "xyz.openbmc_project.Inventory.Decorator.Asset": {
-        "Manufacturer": "$PRODUCT_MANUFACTURER",
-        "Model": "$PRODUCT_PRODUCT_NAME",
-        "PartNumber": "$PRODUCT_PART_NUMBER",
-        "SerialNumber": "$PRODUCT_SERIAL_NUMBER",
-        "SparePartNumber": "$BOARD_INFO_AM1"
-    },
-    "xyz.openbmc_project.Inventory.Decorator.AssetTag": {
-        "AssetTag": "$PRODUCT_ASSET_TAG"
-    },
-    "xyz.openbmc_project.Inventory.Decorator.Revision": {
-        "Version": "$PRODUCT_VERSION"
-    }
-}
diff --git a/configurations/yosemite4_fanboard_fsc_max_adc_ti.json b/configurations/yosemite4_fanboard_fsc_max_adc_ti.json
index 3a29891..c8f8ff0 100644
--- a/configurations/yosemite4_fanboard_fsc_max_adc_ti.json
+++ b/configurations/yosemite4_fanboard_fsc_max_adc_ti.json
@@ -683,8 +683,8 @@
             "Type": "I2CFan"
         }
     ],
-    "Name": "Yosemite 4 FAN Board $bus % 30",
-    "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'Fan Board', 'PRODUCT_PRODUCT_NAME': 'Yosemite V4'})",
+    "Name": "Yosemite 4 Fan Board $bus % 30",
+    "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'FAN Board FSC-MAX ADC-TI', 'PRODUCT_PRODUCT_NAME': 'Yosemite V4'})",
     "Type": "Board",
     "xyz.openbmc_project.Inventory.Decorator.Asset": {
         "Manufacturer": "$PRODUCT_MANUFACTURER",
diff --git a/configurations/yosemite4_fanboard_fsc_nct.json b/configurations/yosemite4_fanboard_fsc_nct.json
deleted file mode 100644
index be0bc2d..0000000
--- a/configurations/yosemite4_fanboard_fsc_nct.json
+++ /dev/null
@@ -1,889 +0,0 @@
-{
-    "Exposes": [
-        {
-            "Address": "$address",
-            "Bus": "$bus",
-            "Name": "Fan Board $bus % 30 FRU",
-            "Type": "EEPROM"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN0_TACH_IL",
-                "Pwm": 5,
-                "PwmName": "FANBOARD0_FAN0_PWM",
-                "Tachs": [
-                    1
-                ]
-            },
-            "Index": 1,
-            "Name": "FANBOARD0_FAN0_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN0_TACH_OL",
-                "Pwm": 5,
-                "PwmName": "FANBOARD0_FAN0_PWM",
-                "Tachs": [
-                    2
-                ]
-            },
-            "Index": 2,
-            "Name": "FANBOARD0_FAN0_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN1_TACH_IL",
-                "Pwm": 5,
-                "PwmName": "FANBOARD0_FAN1_PWM",
-                "Tachs": [
-                    1
-                ]
-            },
-            "Index": 1,
-            "Name": "FANBOARD0_FAN1_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN1_TACH_OL",
-                "Pwm": 5,
-                "PwmName": "FANBOARD0_FAN1_PWM",
-                "Tachs": [
-                    2
-                ]
-            },
-            "Index": 2,
-            "Name": "FANBOARD0_FAN1_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN2_TACH_IL",
-                "Pwm": 5,
-                "PwmName": "FANBOARD1_FAN2_PWM",
-                "Tachs": [
-                    1
-                ]
-            },
-            "Index": 1,
-            "Name": "FANBOARD1_FAN2_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN2_TACH_OL",
-                "Pwm": 5,
-                "PwmName": "FANBOARD1_FAN2_PWM",
-                "Tachs": [
-                    2
-                ]
-            },
-            "Index": 2,
-            "Name": "FANBOARD1_FAN2_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN3_TACH_IL",
-                "Pwm": 5,
-                "PwmName": "FANBOARD1_FAN3_PWM",
-                "Tachs": [
-                    1
-                ]
-            },
-            "Index": 1,
-            "Name": "FANBOARD1_FAN3_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN3_TACH_OL",
-                "Pwm": 5,
-                "PwmName": "FANBOARD1_FAN3_PWM",
-                "Tachs": [
-                    2
-                ]
-            },
-            "Index": 2,
-            "Name": "FANBOARD1_FAN3_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN4_TACH_IL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD0_FAN4_PWM",
-                "Tachs": [
-                    14
-                ]
-            },
-            "Index": 14,
-            "Name": "FANBOARD0_FAN4_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN4_TACH_OL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD0_FAN4_PWM",
-                "Tachs": [
-                    0
-                ]
-            },
-            "Index": 0,
-            "Name": "FANBOARD0_FAN4_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN5_TACH_IL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD0_FAN5_PWM",
-                "Tachs": [
-                    14
-                ]
-            },
-            "Index": 14,
-            "Name": "FANBOARD0_FAN5_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN5_TACH_OL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD0_FAN5_PWM",
-                "Tachs": [
-                    0
-                ]
-            },
-            "Index": 0,
-            "Name": "FANBOARD0_FAN5_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN6_TACH_IL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD1_FAN6_PWM",
-                "Tachs": [
-                    14
-                ]
-            },
-            "Index": 14,
-            "Name": "FANBOARD1_FAN6_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN6_TACH_OL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD1_FAN6_PWM",
-                "Tachs": [
-                    0
-                ]
-            },
-            "Index": 0,
-            "Name": "FANBOARD1_FAN6_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN7_TACH_IL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD1_FAN7_PWM",
-                "Tachs": [
-                    14
-                ]
-            },
-            "Index": 14,
-            "Name": "FANBOARD1_FAN7_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN7_TACH_OL",
-                "Pwm": 2,
-                "PwmName": "FANBOARD1_FAN7_PWM",
-                "Tachs": [
-                    0
-                ]
-            },
-            "Index": 0,
-            "Name": "FANBOARD1_FAN7_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN8_TACH_IL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD0_FAN8_PWM",
-                "Tachs": [
-                    9
-                ]
-            },
-            "Index": 9,
-            "Name": "FANBOARD0_FAN8_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN8_TACH_OL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD0_FAN8_PWM",
-                "Tachs": [
-                    12
-                ]
-            },
-            "Index": 12,
-            "Name": "FANBOARD0_FAN8_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN9_TACH_IL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD0_FAN9_PWM",
-                "Tachs": [
-                    9
-                ]
-            },
-            "Index": 9,
-            "Name": "FANBOARD0_FAN9_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 30,
-            "Connector": {
-                "Name": "FANBOARD0_FAN9_TACH_OL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD0_FAN9_PWM",
-                "Tachs": [
-                    12
-                ]
-            },
-            "Index": 12,
-            "Name": "FANBOARD0_FAN9_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN10_TACH_IL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD1_FAN10_PWM",
-                "Tachs": [
-                    9
-                ]
-            },
-            "Index": 9,
-            "Name": "FANBOARD1_FAN10_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x21",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN10_TACH_OL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD1_FAN10_PWM",
-                "Tachs": [
-                    12
-                ]
-            },
-            "Index": 12,
-            "Name": "FANBOARD1_FAN10_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN11_TACH_IL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD1_FAN11_PWM",
-                "Tachs": [
-                    9
-                ]
-            },
-            "Index": 9,
-            "Name": "FANBOARD1_FAN11_TACH_IL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 17380
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 13140
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        },
-        {
-            "Address": "0x23",
-            "Bus": 31,
-            "Connector": {
-                "Name": "FANBOARD1_FAN11_TACH_OL",
-                "Pwm": 0,
-                "PwmName": "FANBOARD1_FAN11_PWM",
-                "Tachs": [
-                    12
-                ]
-            },
-            "Index": 12,
-            "Name": "FANBOARD1_FAN11_TACH_OL",
-            "PowerState": "Always",
-            "Thresholds": [
-                {
-                    "Direction": "greater than",
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 15950
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 11970
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 1000
-                }
-            ],
-            "Type": "I2CFan"
-        }
-    ],
-    "Name": "Yosemite 4 Fan Board $bus % 30 FSC",
-    "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'FAN Board FSC-NCT ADC-.*', 'PRODUCT_PRODUCT_NAME': 'Yosemite V4'})",
-    "Type": "Board",
-    "xyz.openbmc_project.Inventory.Decorator.Asset": {
-        "Manufacturer": "$PRODUCT_MANUFACTURER",
-        "Model": "$PRODUCT_PRODUCT_NAME",
-        "PartNumber": "$PRODUCT_PART_NUMBER",
-        "SerialNumber": "$PRODUCT_SERIAL_NUMBER"
-    },
-    "xyz.openbmc_project.Inventory.Decorator.AssetTag": {
-        "AssetTag": "$PRODUCT_ASSET_TAG"
-    },
-    "xyz.openbmc_project.Inventory.Decorator.Revision": {
-        "Version": "$PRODUCT_VERSION"
-    }
-}
diff --git a/meson.build b/meson.build
index c47294b..83ae345 100644
--- a/meson.build
+++ b/meson.build
@@ -193,11 +193,8 @@ configs = [
     'vegman_rx20_baseboard.json',
     'vegman_sx20_baseboard.json',
     'wft_baseboard.json',
-    'yosemite4_fanboard_adc_ti.json',
     'yosemite4_fanboard_fsc_max_adc_ti.json',
     'yosemite4_fanboard_fsc_nct_adc_max.json',
-    'yosemite4_fanboard_fsc_max.json',
-    'yosemite4_fanboard_fsc_nct.json',
     'yosemite4_sentineldome_t1.json',
     'yosemite4_sentineldome_t1_retimer.json',
     'yosemite4_sentineldome_t2.json',
-- 
2.25.1

